adicionei base de dados em Alagoas

MariaCarolinass opened the pull request:
### Título

### Este PR faz o seguinte:

- [X] Inclui uma nova base de dados
- [ ] Inclui nova funcionalidade
- [ ] Altera o layout
- [ ] Formatação ou refactor de código
- [ ] Resolve a issue#{id}

---

### Se há mudança de layout ou funcionaliade, foi testado em quais browsers:

Mobile

- Android
  - [ ] Chrome
- iOS
  - [ ] Chrome
  - [ ] Safari

Desktop

- Windows
  - [ ] Chrome
  - [ ] Firefox
- MacOs
  - [ ] Chrome
  - [ ] Firefox
  - [ ] Safari
- Linux
  - [ ] Chrome
  - [ ] Firefox

---

# Descrição
Adicionei base de dados em Alagoas. Foram adicionados alguns municípios e links do Portal da Transparência da cidade e da câmara de vereadores

MariaCarolinass committed:
adicionei base de dados em Alagoas

atarrisse commented on _states/alagoas.md:
```diff
@@ -3,6 +3,58 @@ layout: default
+### Atalaia
+
+- **[Portal da Transparência do Município de Atalaia](http://transparencia.atalaia.al.gov.br/)**: http://transparencia.atalaia.al.gov.br/
+- **[Portal da Transparência da Câmara Municipal de Atalaia](http://www.atalaia.al.leg.br/)**: http://www.atalaia.al.leg.br/
```
Esse endereço é a home da Câmara Municipal. Você pode atualizar pro endereço do portal, por favor?

atarrisse commented on _states/alagoas.md:
```diff
@@ -3,6 +3,58 @@ layout: default
+### Batalha
+
+- **[Portal da Transparência do Município de Batalha](http://www.batalha.al.gov.br/sic-batalha/)**: http://www.batalha.al.gov.br/sic-batalha/
+- **[Portal da Transparência da Câmara Municipal de Batalha](http://www.batalha.al.leg.br/)**: http://www.batalha.al.leg.br/
```
Esse endereço é a home da Câmara Municipal. Você pode atualizar pro endereço do portal, por favor?

atarrisse commented on _states/alagoas.md:
```diff
@@ -3,6 +3,58 @@ layout: default
+### Belém
+
+- **[Portal da Transparência do Município de Belém](https://belem.al.gov.br/portaltransparencia)**: https://belem.al.gov.br/portaltransparencia
+- **[Portal da Transparência da Câmara Municipal de Belém](https://belem.al.gov.br/)**: https://belem.al.gov.br/
```
Esse site é da Prefeitura de Belém. O portal da transparência deles é o link que vc já colocou antes desse. Não precisamos desse segundo :)

atarrisse commented on _states/alagoas.md:
```diff
@@ -3,6 +3,58 @@ layout: default
+
+### Branquinha
+
+- **[Portal da Transparência do Município de Branquinha](http://5.172.159.218/branquinha/index.php?route=home&token=hdcw25a0km1s2wd3x1qq5x01a)**: http://5.172.159.218/branquinha/index.php?route=home&token=hdcw25a0km1s2wd3x1qq5x01a
```
Existe um link mais amigável pro Portal da Transparência de Branquinha: `http://www.iaplicativo.com/branquinha/`

atarrisse commented on _states/alagoas.md:
```diff
@@ -3,6 +3,58 @@ layout: default
+### Branquinha
+
+- **[Portal da Transparência do Município de Branquinha](http://5.172.159.218/branquinha/index.php?route=home&token=hdcw25a0km1s2wd3x1qq5x01a)**: http://5.172.159.218/branquinha/index.php?route=home&token=hdcw25a0km1s2wd3x1qq5x01a
+- **[Portal da Transparência da Câmara Municipal de Branquinha](http://branquinha.al.gov.br/home/)**: http://branquinha.al.gov.br/home/
```
Esse é o site da prefeitura de Branquinha. O portal da transparência deles é o link que vc já colocou antes desse. Não precisamos desse segundo :)

atarrisse commented on _states/alagoas.md:
```diff
@@ -3,6 +3,58 @@ layout: default
+### Cacimbinhas
+
+- **[Portal da Transparência do Município de Cacimbinhas](https://transparencia.cacimbinhas.al.gov.br/)**: https://transparencia.cacimbinhas.al.gov.br/
+- **[Portal da Transparência da Câmara Municipal de Cacimbinhas](http://www.cacimbinhas.al.leg.br/)**: http://www.cacimbinhas.al.leg.br/
```
Esse endereço é a home da Câmara Municipal. Você pode atualizar pro endereço do portal, por favor?

atarrisse commented on _states/alagoas.md:
```diff
@@ -3,6 +3,58 @@ layout: default
+### Cajueiro
+
+- **[Portal da Transparência do Município de Cajueiro](http://www.cajueiro.al.gov.br/transparencia/)**: http://www.cajueiro.al.gov.br/transparencia/
+- **[Portal da Transparência da Câmara Municipal de Cajueiro](https://www.cajueiro.al.leg.br/)**: https://www.cajueiro.al.leg.br/
```
Esse endereço é a home da Câmara Municipal. Você pode atualizar pro endereço do portal, por favor?

atarrisse reviewed (changes requested):
Oi Carol, muito obrigada pela sua contribuição <3
Eu coloquei comentários em alguns links.
Se você tiver dúvidas, é só entrar em contato por aqui ou pelo grupo do telegram :)

MariaCarolinass committed:
fiz alterações recomendadas pela colaboradados

Sobre o Portal da Transparência da Câmara Municipal de Arapiraca, Atalaia e Barra de São Miguel, eu não consegui encontrar outro link, só esses links que estavam aí.

MariaCarolinass committed:
Atualizei lista ul de Arapiraca

Os dois itens da ul estavam em uma linha só

MariaCarolinass commented:
Acabei de atualizar as alterações em que pediu

atarrisse reviewed (approved):
Muito obrigada, Carol!
Feliz Hacktoberfest 🍺
